File: CRS-Group-B-Spring-Rest-JDBCTemplate/src/main/java/com/lti/dto/RegisteredCourse.java
package com.lti.dto;

import java.io.Serializable;

public class RegisteredCourse implements Serializable{

	private Student student;
	private Course course;
	private boolean feePaid;
	private double grade;
	
	public Student getStudent() {
		return student;
	}
	public void setStudent(Student student) {
		this.student = student;
	}
	public Course getCourse() {
		return course;
	}
	public void setCourse(Course course) {
		this.course = course;
	}
	public boolean isFeePaid() {
		return feePaid;
	}
	public void setFeePaid(boolean feePaid) {
		this.feePaid = feePaid;
	}
	public double getGrade() {
		return grade;
	}
	public void setGrade(double grade) {
		this.grade = grade;
	}
	
	
}

File: CRS-Group-B-UI-UX/CRS-Group-B-UI-UX/src/app/model/registered-course.ts
import { Course } from "./course";
import { Student } from "./student";

export class RegisteredCourse {
    public course: Course;
    public student: Student;
    public feePaid: boolean;
    public grade: number;

    constructor(course: Course, student: Student){
        this.course = course;
        this.student = student;
        this.feePaid = false;
        this.grade = -1;
    }
}
